tests/utils/verification/volumeproperties.go

```diff
@@ -21,32 +21,33 @@ package verification
 import (
 	"log"
 	"strings"
-	"time"
 
 	"github.com/vmware/docker-volume-vsphere/tests/constants/admincli"
 	"github.com/vmware/docker-volume-vsphere/tests/constants/dockercli"
 	"github.com/vmware/docker-volume-vsphere/tests/constants/properties"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/misc"
 	"github.com/vmware/docker-volume-vsphere/tests/utils/ssh"
 )
 
 const (
-	pollTimeout = 64
+	pollTimeout = 60
 )
 
 // GetVMAttachedToVolUsingDockerCli returns attached to vm field of volume using docker cli
+// TODO: make this private member after finishing refactoring of volmprop_test.go and remove this TODO
 func GetVMAttachedToVolUsingDockerCli(volName string, hostname string) string {
 	cmd := dockercli.InspectVolume + " --format '{{index .Status \"attached to VM\"}}' " + volName
-	op := ExecCmd(hostname, cmd)
+	op, _ := ssh.InvokeCommand(hostname, cmd)
 	if op == "" {
 		log.Fatal("Null value is returned by docker cli when looking for attached to vm field for volume. Output: ", op)
 	}
-	return strings.TrimSpace(op)
+	return op
 }
 
 // GetVMAttachedToVolUsingAdminCli returns attached to vm field of volume using admin cli
 func GetVMAttachedToVolUsingAdminCli(volName string, hostname string) string {
 	cmd := admincli.ListVolumes + "-c volume,attached-to 2>/dev/null | grep " + volName
-	op := ExecCmd(hostname, cmd)
+	op, _ := ssh.InvokeCommand(hostname, cmd)
```

Comment:
Check for error and then output? the command may be deprecated or incorrect syntax.